ouroboros-network/test/Test/Ouroboros/Network/PeerSelection/PeerMetric.hs

```diff
@@ -293,6 +294,12 @@ prop_insert_peer script =
o 46),FetchedHeader (TestAddress 12) (SlotNo 52),FetchedBlock (TestAddress 2) (SlotNo 54) 1,FetchedHeader (TestAddress 1) (SlotNo 54),FetchedBlock (TestAddress 8) (SlotNo 57) 1,FetchedBlock (TestAddress 3) (SlotNo 57) 1,FetchedHeader (TestAddress 10) (SlotNo 60),FetchedBlock (TestAddress 1) (SlotNo 60) 1,FetchedHeader (TestAddress 4) (SlotNo 61),FetchedHeader (TestAddress 3) (SlotNo 62),FetchedHeader (TestAddress 4) (SlotNo 62),FetchedHeader (TestAddress 7) (SlotNo 65),FetchedBlock (TestAddress 1) (SlotNo 65) 1,FetchedBlock (TestAddress 9) (SlotNo 66) 1,FetchedBlock (TestAddress 1) (SlotNo 66) 1,FetchedBlock (TestAddress 1) (SlotNo 70) 1,FetchedHeader (TestAddress 12) (SlotNo 71),FetchedHeader (TestAddress 1) (SlotNo 73),FetchedBlock (TestAddress 1) (SlotNo 73) 1,FetchedHeader (TestAddress 11) (SlotNo 76),FetchedBlock (TestAddress 4) (SlotNo 80) 1,FetchedHeader (TestAddress 9) (SlotNo 81),FetchedHeader (TestAddress 7) (SlotNo 83),FetchedHeader (TestAddress 1) (SlotNo 87),FetchedBlock (TestAddress 13) (SlotNo 90) 1,FetchedBlock (TestAddress 5) (SlotNo 93) 1,FetchedHeader (TestAddress 12) (SlotNo 96),FetchedBlock (TestAddress 8) (SlotNo 97) 1,FetchedBlock (TestAddress 9) (SlotNo 99) 1,FetchedBlock (TestAddress 10) (SlotNo 100) 1,FetchedBlock (TestAddress 3) (SlotNo 100) 1,FetchedBlock (TestAddress 1) (SlotNo 101) 1,FetchedBlock (TestAddress 6) (SlotNo 101) 1,FetchedBlock (TestAddress 1) (SlotNo 103) 1,FetchedHeader (TestAddress 9) (SlotNo 103),FetchedHeader (TestAddress 10) (SlotNo 105),FetchedHeader (TestAddress 2) (SlotNo 105),FetchedBlock (TestAddress 9) (SlotNo 108) 1,FetchedBlock (TestAddress 5) (SlotNo 108) 1,FetchedHeader (TestAddress 12) (SlotNo 111),FetchedBlock (TestAddress 1) (SlotNo 111) 1,FetchedHeader (TestAddress 8) (SlotNo 111),FetchedBlock (TestAddress 10) (SlotNo 118) 1,FetchedBlock (TestAddress 15) (SlotNo 121) 1,FetchedBlock (TestAddress 7) (SlotNo 122) 1,FetchedBlock (TestAddress 9) (SlotNo 126) 1,FetchedBlock (TestAddress 4) (SlotNo 129) 1,FetchedHeader (TestAddress 3) (SlotNo 131),FetchedBlock (TestAddress 5) (SlotNo 131) 1,FetchedHeader (TestAddress 2) (SlotNo 132),FetchedHeader (TestAddress 4) (SlotNo 136),FetchedBlock (TestAddress 3) (SlotNo 137) 1,FetchedBlock (TestAddress 17) (SlotNo 137) 1,FetchedHeader (TestAddress 8) (SlotNo 138),FetchedHeader (TestAddress 1) (SlotNo 138),FetchedHeader (TestAddress 12) (SlotNo 138),FetchedHeader (TestAddress 3) (SlotNo 139),FetchedHeader (TestAddress 11) (SlotNo 141),FetchedHeader (TestAddress 2) (SlotNo 141),FetchedHeader (TestAddress 14) (SlotNo 141),FetchedHeader (TestAddress 1) (SlotNo 142),FetchedBlock (TestAddress 9) (SlotNo 144) 1,FetchedBlock (TestAddress 11) (SlotNo 144) 1,FetchedBlock (TestAddress 2) (SlotNo 145) 1,FetchedHeader (TestAddress 13) (SlotNo 150),FetchedHeader (TestAddress 2) (SlotNo 152),FetchedHeader (TestAddress 6) (SlotNo 152),FetchedBlock (TestAddress 2) (SlotNo 152) 1,FetchedBlock (TestAddress 3) (SlotNo 152) 1,FetchedBlock (TestAddress 6) (SlotNo 154) 1])}
```

Comment:
Could you pretty print this script.  It will payoff once we will be updateing one of the types involved.

Reply:
Can do!